core/templates/pages/contributor-dashboard-page/services/contribution-and-review.service.ts

```diff
@@ -119,6 +122,20 @@ angular.module('oppia').factory('ContributionAndReviewService', [
         }).then(function() {
           onSuccess(suggestionId);
         }, () => onFailure && onFailure(suggestionId));
+      },
+      updateTranslation: function(
+          suggestionId, translationHtml,
+          onSuccess, onFailure) {
+        var url = UrlInterpolationService.interpolateUrl(
+          _UPDATE_TRANSLATION_HANDLER_URL, {
+            suggestion_id: suggestionId
+          });
+        const putData = {
```

Comment:
This seems simple enough to declare inline below.

Reply:
Done